ci/dev/test-e2e.sh

```diff
@@ -6,7 +6,7 @@ main() {
   cd test
   # We set these environment variables because they're used in the e2e tests
   # they don't have to be these values, but these are the defaults
-  PASSWORD=e45432jklfdsab CODE_SERVER_ADDRESS=http://localhost:8080 yarn folio --config=config.ts --reporter=list "$@"
+  PASSWORD=e45432jklfdsab CODE_SERVER_ADDRESS=http://localhost:8080 yarn playwright test "$@"
```

Comment:
I removed the two parameters there, to have all Playwright related configuration in its config file. Also `playwright.config.ts` is easier to understand than `config.ts`

Reply:
Agreed! `playwright.config.ts` is easier to understand. Good call!